src/utils.ts

```diff
@@ -0,0 +1,43 @@
+import { Contract, constants } from 'ethers'
+
+import { Provider } from '@ethersproject/providers'
+
+/**
+ * Returns the broker address from the Celo registry
+ * @param provider an ethers provider
+ * @returns the broker address
+ */
+export async function getBrokerAddressFromRegistry(
+  provider: Provider
+): Promise<string> {
+  const celoRegistryAddress = '0x000000000000000000000000000000000000ce10'
+  const brokerIdentifier = 'Broker'
+
+  const registryAbi = [
+    'function getAddressForString(string calldata identifier) external view returns (address)',
+  ]
+  const contract = new Contract(celoRegistryAddress, registryAbi, provider)
+
+  const brokerAddress = await contract.getAddressForString(brokerIdentifier)
+  if (brokerAddress === constants.AddressZero) {
```

Comment:
consider also checking if broker is not a valid address to cover more cases here.
e.g.
`if (!isValidAddress(brokerAddress) || brokerAddress === constants.AddressZero)`

Relevant: https://github.com/mento-protocol/mento-web/blob/main/src/utils/addresses.ts#L4

Reply:
Hmm I don't think this validation would add much value here given that the address is read from the on-chain registry contract. If it was somehow set to an invalid address the whole deployment would be broken. wdyt?